src/main/java/org/tron/core/net/node/NodeDelegateImpl.java

```diff
@@ -116,7 +116,8 @@ public void handleTransaction(TransactionCapsule trx) throws BadTransactionExcep
 
     //todo: limit the count of block to send peer by one time.
     long unForkedBlockIdNum = unForkedBlockId.getNum();
-    long len = Longs.min(dbManager.getHeadBlockNum(), unForkedBlockIdNum +NodeConstant.SYNC_FETCH_BATCH_NUM);
+    long len = Longs
+        .min(dbManager.getHeadBlockNum(), unForkedBlockIdNum + NodeConstant.SYNC_FETCH_BATCH_NUM);
```

Comment:
Not sure if this break is done by a formatter or accidental. Please check

Reply:
this may be  formatted by ide.